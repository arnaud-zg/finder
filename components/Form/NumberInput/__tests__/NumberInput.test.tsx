import { Formik } from 'formik'
import React from 'react'
import renderer from 'react-test-renderer'
import { NumberInput } from '../NumberInput'

describe('NumberInput', () => {
  it('renders correctly', () => {
    const testRenderer = renderer.create(
      <Formik
        initialValues={{ phone: '' }}
        onSubmit={() => {}}
        render={() => <NumberInput name="phone" />}
      />
    )
    expect(testRenderer.toJSON()).toMatchSnapshot()
  })
})
