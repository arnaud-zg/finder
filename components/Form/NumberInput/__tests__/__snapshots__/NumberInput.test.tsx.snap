// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NumberInput renders correctly 1`] = `
<input
  className="f13jo2op"
  name="phone"
  onBlur={[Function]}
  onChange={[Function]}
  type="number"
  value=""
/>
`;
